hack/make-rules/test-cmd.sh

```diff
@@ -40,16 +40,16 @@ function run_kube_apiserver() {
   # Include RBAC (to exercise bootstrapping), and AlwaysAllow to allow all actions
   AUTHORIZATION_MODE="RBAC,AlwaysAllow"
 
-  # Enable features
-  ENABLE_FEATURE_GATES="DryRun=true"
+  # Features setting
+  FEATURE_GATES="DryRun=true,TokenRequest=false,TokenRequestProjection=false"
```

Comment:
why was this change required?

Reply:
Without this change, error would happen in 
https://github.com/kubernetes/kubernetes/blob/8012b9583e5301c2f66db4118b38433d035b87c4/test/cmd/get.sh#L134
https://github.com/kubernetes/kubernetes/blob/8012b9583e5301c2f66db4118b38433d035b87c4/test/cmd/core.sh#L28
Since test-cmd.sh is mainly test the function of kubectl, I think just disable TokenRequest feature to avoid this is reasonable.

Reply:
if the apiserver isn't started with a private key for generating tokens (which this one is not), it should not serve the token request subresource, and the serviceaccount admission plugin should not inject tokenprojection volumes. we should not need to flip the feature gates differently here, but should instead make sure these features will work properly as the feature gates are promoted. the test you reference ("no configmaps exist in the namespace") is a fragile test that should probably be changed to more specifically test the precondition we actually care about.